test/system/rmi.bats

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bats
+
+load 'libs/bats-support/load'
+load 'libs/bats-assert/load'
+load 'libs/helpers'
+
+setup() {
+  cleanup_all
+}
+
+teardown() {
+  cleanup_all
+}
+
+
+@test "rmi: Remove all images with the default image present" {
+  num_of_images=$(list_images)
+  assert [ $num_of_images -eq 0 ]
```

Comment:
It didn't have the assert because the `setup()` function makes sure that the all the images are removed before each test, so this always be 0.
Although, it's probably good idea (for clarity and consistency sake) to add the assert, thanks.

One thing, I do think it's better to use `assert_equal` instead, as @hexagonrecursion suggested. It's a bit clearer and more consistent with the other asserts.
```bash
asser_equal "$num_of_images" 0
```